rakelib/examples.rake

```diff
@@ -0,0 +1,107 @@
+# Defines tasks related to generating example apps using the gem's generator.
+# Allows us to create and test apps generated using a wide range of options.
+#
+# Also see example_type.rb
+require "yaml"
+
+require_relative "example_type"
+require_relative "task_helpers"
+
+include ReactOnRails::TaskHelpers
+
+namespace :examples do
+  # Loads data from examples_config.yml and instantiates corresponding ExampleType objects
+  examples_config_file = File.expand_path("../examples_config.yml", __FILE__)
+  examples_config = YAML.load(File.read(examples_config_file))
+  examples_config["example_type_data"].each do |example_type_data|
+    example_type_data.keys.each { |key| example_type_data[key.to_sym] = example_type_data.delete(key) }
```

Comment:
well actually this fails in the linter, and I think it would be weird to randomly have activesupport in our linters docker image. I think I'm just going to keep it as is.